docs/source/upgrade.rst

```diff
@@ -25,6 +25,14 @@ by CCCL (currently CUDA 11 & 12), but this change leads to the following consequ
 As a result of this movement, CuPy now follows the same compiler requirement as CCCL (and, in turn, CUDA Toolkit) and requires C++11 as
 the lowest C++ standard. CCCL expects to move to C++17 in the near future.
 
+Change in :func:`cupy.asnumpy`/:meth:`cupy.ndarray.get` Behavior
+----------------------------------------------------------------
+
+When transferring a CuPy array from GPU to CPU (as a NumPy array), previously the transfer could be nonblocking and not properly ordered,
+leading to potential data race if the resulting array is modified on host immediately after the copy starts. In CuPy v13, the default
```

Comment:
Could it be clarified that only people using user-defiend streams have been affected by this "data race"?

Reply:
Done in 93d9c54.